File: SymfonyApplication/src/Entity/Societe.php
<?php

namespace App\Entity;

use App\Repository\SocieteRepository;
use Doctrine\ORM\Mapping as ORM;

/**
 * @ORM\Entity(repositoryClass=SocieteRepository::class)
 */
class Societe
{
    /**
     * @ORM\Id
     * @ORM\GeneratedValue
     * @ORM\Column(type="integer")
     */
    private $id;

    /**
     * @Assert\NotBlank(message="le champs Numero de registre est obligatoire * ")
     * @ORM\Column(type="string", length=50)
     */
    private $numregistre;

    /**
     * @Assert\NotBlank(message="le champs adresse est obligatoire * ")
     * @ORM\Column(type="string", length=30)
     */
    private $adresse;

    /**
     * @Assert\NotBlank(message="le champs adresse est obligatoire * ")
     * @ORM\Column(type="string", length=20)
     */
    private $type;

    /**
     * @ORM\Column(type="boolean")
     */
    private $etat;

    /**
     * @ORM\OneToOne(targetEntity=User::class, inversedBy="Societe", cascade={"persist", "remove"})
     * @ORM\JoinColumn(nullable=false)
     */
    private $Useraccount;

    /**
     * @ORM\Column(type="string", length=20)
     */
    private $nom;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getNumregistre(): ?string
    {
        return $this->numregistre;
    }

    public function setNumregistre(string $numregistre): self
    {
        $this->numregistre = $numregistre;

        return $this;
    }

    public function getAdresse(): ?string
    {
        return $this->adresse;
    }

    public function setAdresse(string $adresse): self
    {
        $this->adresse = $adresse;

        return $this;
    }

    public function getType(): ?string
    {
        return $this->type;
    }

    public function setType(string $type): self
    {
        $this->type = $type;

        return $this;
    }

    public function getEtat(): ?bool
    {
        return $this->etat;
    }

    public function setEtat(bool $etat): self
    {
        $this->etat = $etat;

        return $this;
    }

    public function getUseraccount(): ?User
    {
        return $this->Useraccount;
    }

    public function setUseraccount(User $Useraccount): self
    {
        $this->Useraccount = $Useraccount;

        return $this;
    }

    public function getNom(): ?string
    {
        return $this->nom;
    }

    public function setNom(string $nom): self
    {
        $this->nom = $nom;

        return $this;
    }
}

File: MobileApplication/src/HolidaysHiatus/entities/Societe.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package HolidaysHiatus.entities;

/**
 *
 * @author ASUS
 */
public class Societe {
    int id;
    int useraccount_id;
    String numregistre;
    String adresse;
    String type;
    boolean etat;
    String nom;

    public Societe(int useraccount_id, String numregistre, String adresse, String type, boolean etat, String nom) {
        this.useraccount_id = useraccount_id;
        this.numregistre = numregistre;
        this.adresse = adresse;
        this.type = type;
        this.etat = etat;
        this.nom = nom;
    }

    public Societe(int id, int useraccount_id, String numregistre, String adresse, String type, boolean etat, String nom) {
        this.id = id;
        this.useraccount_id = useraccount_id;
        this.numregistre = numregistre;
        this.adresse = adresse;
        this.type = type;
        this.etat = etat;
        this.nom = nom;
    }

    public Societe() {
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getUseraccount_id() {
        return useraccount_id;
    }

    public void setUseraccount_id(int useraccount_id) {
        this.useraccount_id = useraccount_id;
    }

    public String getNumregistre() {
        return numregistre;
    }

    public void setNumregistre(String numregistre) {
        this.numregistre = numregistre;
    }

    public String getAdresse() {
        return adresse;
    }

    public void setAdresse(String adresse) {
        this.adresse = adresse;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public boolean isEtat() {
        return etat;
    }

    public void setEtat(boolean etat) {
        this.etat = etat;
    }

    public String getNom() {
        return nom;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    @Override
    public String toString() {
        return "Societe{" + "id=" + id + ", useraccount_id=" + useraccount_id + ", numregistre=" + numregistre + ", adresse=" + adresse + ", type=" + type + ", etat=" + etat + ", nom=" + nom + '}';
    }
      public void setEtat(String etat) {
       if(etat.compareTo("true")==0||etat.compareTo("1")==0){
       this.etat=true;
       }else if(etat.compareTo("false")==0||etat.compareTo("0")==0){
       this.etat=false;
       }
    }
    

}